README.md

```diff
@@ -139,13 +139,8 @@ There is `Vagrantfile` provided with braid, that will setup a staging sever.
 It uses the address `172.16.255.140`, and there is a braid config named `vagrant` that connects to it by default.
 
 ```shell
-# Get the required base OS image if you don't have it yet.
-vagrant box add precise64 http://files.vagrantup.com/precise64.box
```

Comment:
Your vagrant is very old -- 1.7.3 is out, and uses atlas fine (which hosts these boxes).

Also, the ssh-agent stuff is kinda irrelevant, since it doesn't use the vagrant insecure key after this patch.